released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "dev@clarus.me"
+homepage: "https://github.com/clarus/coq-hello-world"
+dev-repo: "git+https://github.com/clarus/coq-hello-world.git"
+bug-reports: "https://github.com/clarus/coq-hello-world/issues"
+authors: ["Guillaume Claret"]
+license: "MIT"
+build: [
+  ["./configure.sh"]
+  [make "-j%{jobs}%"]
+  [make "-C" "extraction"]
+]
+install: [
+  ["install" "-T" "extraction/main.native" "%{bin}%/helloWorld"]
+]
+depends: [
```

Comment:
Just to be clear, this package doesn't require Coq itself in any way (other than perhaps implicitly via dependencies)?

Reply:
Yes it does depend on Coq actually, so I added that explicitly.